db/background_mgr_resync_dcp_test.go

```diff
@@ -520,16 +521,41 @@ function sync(doc, oldDoc){
 	stats := getResyncStats(resyncMgr.Process)
 	assert.Equal(t, int64(2), stats.DocsChanged)
 
+	_, xattrs, _, err := collection.dataStore.GetWithXattrs(ctx, "sgWrite", []string{"$document"})
+	require.NoError(t, err)
+
+	var retrievedVxattr map[string]interface{}
+	require.NoError(t, json.Unmarshal(xattrs["$document"], &retrievedVxattr))
+
+	rev, ok := retrievedVxattr["revid"].(string)
+	assert.True(t, ok, "Unable to retrieve virtual xattr crc32c as string")
```

Comment:
```suggestion
	require.True(t, ok, "Unable to retrieve virtual xattr crc32c as string")
```

since the code will probably panic after.

What does crc32c mean here?